core/src/main/java/com/adobe/granite/translation/connector/bootstrap/core/impl/BootstrapTranslationServiceImpl.java

```diff
@@ -247,7 +241,7 @@ public String createTranslationJob(String name, String description, String strSo
         try {
           SourceFile[] sourceFiles = liltApiClient.getFiles(null);
         } catch (Exception e) {
-          log.warn("error during createTranslationJob {}", e);
+          log.warn("error during createTranslationJob {}", ExceptionUtils.getStackTrace(e));
```

Comment:
maybe we should change all of the new `log.warn` calls to `log.info`. i made them all `warn` because it made it easier to see the messages when i was developing it

Reply:
Sure, I will change info where it makes sense